Test/server/counterexample_commandline.dfy.expect

```diff
@@ -0,0 +1,25 @@
+counterexample_commandline.dfy(29,20): Error: A postcondition might not hold on this return path.
+counterexample_commandline.dfy(17,22): Related location: This is the postcondition that might not hold.
+
+Dafny program verifier finished with 1 verified, 1 error
+Counterexample for first failing assertion: 
+Line 20 character 8: 
```

Comment:
Will we ever get counter examples that attach values to locations in multiple files? It might be clearer and less surprising to print more standard source locations like `counterexample_commandline.dfy(20,8)` instead.